requirements.txt

```diff
@@ -0,0 +1,4 @@
+togeojsontiles == 0.1.2
```

Comment:
I guess spacing is not allowed in dependency name and version. Can you confirm?

Reply:
spacing is required in dependency name and version.
 (https://note.nkmk.me/en/python-pip-install-requirements/)